api-reference/10 UI Widgets/dxDataGrid/1 Configuration/scrolling/mode.md

```diff
@@ -15,7 +15,7 @@ The following scrolling modes are available in the widget:
 A user scrolls a single page only. Scrolling appears only if [all the page's rows](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging/pageSize.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize') do not fit into the widget's height. In this mode, the [pager](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/pager '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/') performs the main navigation and scrolling is auxiliary. If [paging](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/paging '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/') is disabled, the widget loads all rows at once, which may decrease the widget's performance. In this case, we recommend choosing another scrolling mode.
 
 - **Virtual**       
-This mode is an alternative to paging where pages are loaded when they get into the viewport and removed once they leave it. Use this mode if a user should be able to scroll data by pages.
+This mode is an alternative to paging where rows are loaded when they get into the viewport and removed once they leave it. While being loaded and rendered, the rows display gray boxes instead of data due to [Rendering Optimization](/Documentation/Guide/Widgets/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization). Use this mode if a user should be able to scroll data by pages.
```

Comment:
```suggestion
This mode is an alternative to paging where rows are loaded when they get into the viewport and removed once they leave it. Rows display gray boxes instead of data are when they are loaded and rendered due to [Rendering Optimization](/Documentation/Guide/Widgets/DataGrid/Enhance_Performance_on_Large_Datasets/#Rendering_Optimization). Use this mode to allow a user to scroll through pages.
```